inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/WorkloadMetadataDialog.java

```diff
@@ -32,28 +32,23 @@
     private static final long serialVersionUID = 2797336810690526392L;
 
     public WorkloadMetadataDialog(String aID, SourceDocument aDocument,
-                                  List<String> finshedUsers, List<String> inProgressUser)
+                                  List<String> finishedUsers, List<String> inProgressUsers)
     {
         super(aID);
 
-        if (finshedUsers.size() == 0)
-        {
-            finshedUsers.add("-");
+        if (finishedUsers.isEmpty()) {
+            finishedUsers.add("-");
```

Comment:
Please do not modify collections passed to a method without dire need. Put the logic into the "model, e.g.

```
Label userInProgress = new Label("userInProgress", inProgressUsers.isEmpty() ? "-" : inProgressUsers.toString())
```

I think it would be best not to rely on toString() for getting a sane representation of lists - better explicitly join the values into a string using an adequate separator